resources/assets/components/CampaignSingle/index.js

```diff
@@ -55,32 +66,21 @@ class CampaignSingle extends React.Component {
     });
   }
 
-  // Make API call to GET /posts to get posts by filtered status.
-  getPostsByStatus(status, campaignId) {
+  // Make API call to GET /posts to get posts by filtered status and/or tag(s).
+  getPostsByFilter(filters) {
     this.setState({ loadingNewPosts: true });
 
-    this.api.get('posts', {
-      filter: {
-        status: status,
-        campaign_id: campaignId,
-      },
-      include: 'signup,siblings',
-    })
-    .then(json => {
-      this.setState({loadingNewPosts: false });
-      this.props.setNewPosts(json);
-    });
-  }
+    let apiFilter = {
```

Comment:
hmm why do you need to redefine this variable here? It looks like the same object that is passed in?